### ISSUE TITLE
[Bug] [Question] App Icons on the Home Page Are Shown Dimmed For Reasons Unclear

### ISSUE THREAD

**Issue by whitedavidp** [NeoApplications/Neo-Backup#705]
**Description**
On the home page I see some apps with their icons shown dimmed and I do not understand why this is so. So perhaps not a bug but a question...

![Screenshot_20230104-075140](https://user-images.githubusercontent.com/12023355/210614612-632b047d-3c70-4b74-a88c-8abdb4c9a20c.png)

I have tried to figure out the meaning of this. But the closest I have found is on the legend's page where the is an * note. 

![Screenshot_20230104-075123](https://user-images.githubusercontent.com/12023355/210615132-ba765b0e-fbba-4b4d-95cc-020a9facfc09.png)

But that note does not seem to refer to the app's icon but the app type icon. So I don't think this is right either.

**Comment by machiav3lli** [comment]
Nope, it has nothing to do with the note in the help sheet. It's a coil bug.
